ruby/src/layer/wrapper.rb

```diff
@@ -0,0 +1,15 @@
+require 'opentelemetry/sdk'
+require 'opentelemetry/exporter/otlp'
+require 'opentelemetry/instrumentation/aws_lambda'
+
+OpenTelemetry::SDK.configure do |c|
+  c.use 'OpenTelemetry::Instrumentation::AwsLambda'
+end
+
+::OpenTelemetry::Instrumentation.registry.register('OpenTelemetry::Instrumentation::AwsLambda')
+::OpenTelemetry::Instrumentation.registry.install(['OpenTelemetry::Instrumentation::AwsLambda'])
```

Comment:
I'm curious about these two lines. Doesn't `Instrumentation::Base` do this automatically?

Reply:
ah, yes, it should do it automatically, I will remove the registry part.